src/command.js

```diff
@@ -49,26 +49,27 @@ class GraphqurlCommand extends Command {
       queryErrorCb(this, error, queryType, parsedQuery);
     };
 
-    if (queryString === null) {
-      queryString = await executeQueryFromTerminalUI({
-        endpoint: endpoint,
-        headers,
-        variables,
-        name: flags.name,
-      }, successCallback, errorCallback);
-      return;
-    }
-
     const queryOptions = {
-      query: queryString,
-      endpoint: endpoint,
+      endpoint,
```

Comment:
Maybe you want to do this assignment after you check if `queryString` is null.
Assigning here and spreading later is just an extra step IMO.